web-client/src/presenter/computeds/messageDocumentHelper.test.js

```diff
@@ -4,41 +4,41 @@ import { messageDocumentHelper as messageDocumentHeperComputed } from './message
 import { runCompute } from 'cerebral/test';
 import { withAppContextDecorator } from '../../withAppContext';
 
-const { USER_ROLES } = applicationContext.getConstants();
+describe('messageDocumentHelper', () => {
+  const { USER_ROLES } = applicationContext.getConstants();
 
-const messageDocumentHelper = withAppContextDecorator(
-  messageDocumentHeperComputed,
-  applicationContext,
-);
+  const messageDocumentHelper = withAppContextDecorator(
+    messageDocumentHeperComputed,
+    applicationContext,
+  );
 
-const getBaseState = user => {
-  return {
-    permissions: getUserPermissions(user),
+  const getBaseState = user => {
+    return {
+      permissions: getUserPermissions(user),
+    };
+  };
+
+  const docketClerkUser = {
```

Comment:
Nit: I know you just moved these lines, but they can be imported from `/shared/src/test/mockUsers` rather than redefined here, if you'd like to clean this up a bit.

Reply:
Wait, nevermind, that's only on develop. Resolving this!

Reply:
Thank you! Love it!!!